platform/darwin/src/MGLRendererConfiguration.h

```diff
@@ -35,6 +35,19 @@ NS_ASSUME_NONNULL_BEGIN
  which will be available at run time, e.g. "PingFang". */
 @property (nonatomic, readonly) mbgl::optional<std::string> localFontFamilyName;
 
+/**
+ A Boolean value indicating whether symbol layers may enable cross-source symbol
+ collision detection.
+ 
+ Set `MGLCrossSourceCollisions` in your containing app's Info.plist
```

Comment:
Are we sure this is a setting that should be uniform application-wide? Or is there a use case for enabling this setting for one map view but not another in the same application? If the latter, then there needs to be a method on MGLStyle that controls this setting. That means it needs to be possible to pass the setting into `mbgl::Map` at an arbitrary point after the `mbgl::Map` is constructed. (The developer can’t be expected to provide this setting as part of an MGLMapView initializer, because views are typically initialized with nothing but a frame.)

/cc @ChrisLoer

Reply:
If we consider this to be legacy behavior, then perhaps we should name the option somewhat more disparagingly and document it as a legacy behavior.